dev-docs/compliance.md

```diff
@@ -65,9 +67,10 @@ There are several other minor commands not listed here - see `lib/cli.rb` for a
 
 ### login
 
-Saves a credentials file locally. Future invocations of `inspec compliance` use the credentials file to authenticate.
+Saves a credentials file locally. Future invocations of `inspec compliance` or `inspec automate` use the credentials file to authenticate.
```

Comment:
Throughout, list `inspec automate` first. Think of it as being the default command, and compliance as being the backwards compatible alias (even though that is not the implementation).

Reply:
Got it. Will do so. Thanks!